com.unity.visualeffectgraph/Editor/GraphView/Views/VFXView.cs

```diff
@@ -405,7 +408,7 @@ public static VisualTreeAsset LoadUXML(string text)
         public static Texture2D LoadImage(string text)
         {
             string path = string.Format("{0}/VFX/{1}.png", VisualEffectAssetEditorUtility.editorResourcesPath, text);
-            return AssetDatabase.LoadAssetAtPath<Texture2D>(path);
+            return EditorGUIUtility.LoadIcon(path);
```

Comment:
This change has nothing to do with the PR, but while managing icons I discovered that our icons were blurred in the graph because only the low res (16px) versions were used. Now if relevant it uses the @2x version.